### PR TITLE
added shippable CI files; added grunt support

### PR DESCRIPTION
**Pull request by howarddierking**
tests won't pass because they depend on live server/credentials, but otherwise, contains the infrastructure needed for shippable ci

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +26,20 @@
...
     "gulp-bump": "^0.1.11",
     "gulp-jshint": "^1.8.6",
     "gulp-mocha": "^1.1.1",
-    "mocha": "^1.21.4"
+    "grunt": "^0.4.5",
+    "grunt-contrib-jshint": "^0.10.0",
+    "grunt-env": "^0.4.2",
+    "grunt-mocha-test": "^0.12.1",
+    "grunt-npm-install": "^0.1.0",
+    "istanbul": "^0.3.5",
```
Comment (Trudeaucj) [inline package.json]: :+1:

File: test/report.js [hunk 2]
```diff
@@ -48,8 +48,6 @@ describe('Concur Report Tests', function() {
     });
   });
 
-  var reportId;
```
Comment (Trudeaucj) [inline test/report.js]: The test will fail without this. Perhaps it needs a refactor. This gives the ability to use a report ID throughout the tests.

Comment (howarddierking) [inline test/report.js]: Actually, I removed this line because the linter caught it as a duplicate declaration with line 11.

Comment (Trudeaucj) [inline test/report.js]: ahh :shipit:

**Comment by Trudeaucj** [comment]
:shipit:

**Pull request closed without merging**